buildkite/scripts/rosetta-integration-tests.sh

```diff
@@ -250,10 +250,11 @@ if [[ "$MODE" == "full" ]]; then
   send_zkapp_transactions &
 fi
 
-next_block_time=$(mina client status --json | jq '.next_block_production.timing[1].time' | tr -d '"') curr_time=$(date +%s%N | cut -b1-13)
-sleep_time=$((($next_block_time - $curr_time) / 1000))
+mina client status --json
```

Comment:
Is it intentional?

Reply:
yea i thought so, i needed this so often while debugging the test i figured it should be in the log output. I can also remove it if you don't need it. the first step in debugging client output in this test will be to add it back though.